src/events/payload.rs

```diff
@@ -109,6 +109,9 @@ pub enum EventType {
 
     /// A webxdc wants an info message or a changed summary to be notified.
     IncomingWebxdcNotify {
+        /// ID of the chat which the message belongs to.
```

Comment:
```suggestion
        /// ID of the chat which the webxdc instance belongs to.
```
I think this is a clearer wording, particularly when the notification is about a summary change.

Reply:
How about also adding "and the info message"? Because they both are supposed to belong to the same chat, right?

Reply:
A bit awkward, but I'm fine with all the considered options. Feel free to commit.